sdk/core/azure-core-test/pom.xml

```diff
@@ -52,10 +52,76 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
-
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock-standalone</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
```

Comment:
I thought we were trying to remove our dependency on Jetty, why was this added here?

Reply:
Some tests were moved from azure-core to azure-core-test and to run them this is required. When we remove the dependency on Jetty, we have to update this as well.